buzzmobile/sense/maps_querier/maps_querier.py

```diff
@@ -11,7 +11,10 @@
 import googlemaps
 import rospy
 
-import googlemapskey as gmpskey
+try:
+    import googlemapskey as gmpskey
+except ImportError as e:
+    pass
```

Comment:
are you sure we want to fail silently here? Maybe a warning message that says they should init the googlemapskey. If you're ok with that I can push it to this branch.

Reply:
a "This should be a test and you should be faking gmpskey" warning wouldn't be a terrible thing.